src/PrestaShopBundle/Resources/views/Admin/Product/Include/form_combinations.html.twig

```diff
@@ -27,11 +28,28 @@
     </div>
   </div>
 
+  <div id="combinations-bulk-form">
+    <div class="row">
+      <p data-toggle="collapse" href="#bulk-combinations-container" aria-expanded="false" aria-controls="bulk-combinations-container">
+        <b>{{ trans('Bulk actions', {}, 'AdminProducts') }}</b>
+        <i class="material-icons pull-right">keyboard_arrow_down</i>
+      </p>
+      <div class="col-md-12 collapse" style="border:2px solid #bbcdd2;" id="bulk-combinations-container">
```

Comment:
note the design is not part of this pull request, I prefer the inline css here because it will be more easy for @nihco2 to remove it. Do you want me to remove it ?